meteor/client/ui/Settings/ShowStyleBaseSettings.tsx

```diff
@@ -1037,6 +1052,119 @@ const HotkeyLegendSettings = withTranslation()(
 			})
 		}
 
+		onDownloadAHKScript = () => {
```

Comment:
Does it make sense to include this in the public release, considering it uses a hard-coded, private TV2 AHK mapping?

Reply:
I wouldn't say the AHK mapping is hardcoded. It'll remap almost any combo for you. There seems to be only one character specific to Danish layout, which should go away.

Reply:
In the interest of keeping the code-base as unified as possible, we'll allow keeping this. TV2 promises to keep the number of hard-coded things to a minimum :)